backend/spec/requests/api/v1/bugs_spec.rb

```
@@ -0,0 +1,75 @@
+        expect {
+          post "/api/v1/bugs", params: valid_params, headers: headers, as: :json
+        }.to change { Bug.count }.by(1)
+
+        expect(response).to have_http_status(:created)
+        expect(JSON.parse(response.body)["message"]).to eq("メモを保存しました")
+      end
+
+      it "間違った値が入力された場合、バグの登録が失敗し、422エラーを返す" do
+        expect {
+          post "/api/v1/bugs", params: invalid_params, headers: headers, as: :json
+        }.not_to change { Bug.count }
```

RuboCop (1.73)

[warning] Parenthesize the param `change { Bug.count }` to make sure that the block will be associated with the `change` method call.

(Lint/AmbiguousBlockAssociation)